src/main/resources/edu/cornell/kfs/core/documentstore/InstitutionPreferences.json

```diff
@@ -134,8 +137,16 @@
           }
         }
       }, {
-        "businessObjectClass": "org.kuali.kfs.gl.businessobject.CurrentAccountBalance",
-        "linkType": "kfs"
+        "label": "Current Fund Balance",
+        "link": "kr/lookup.do?methodToCall=start&businessObjectClassName=edu.cornell.kfs.gl.businessobject.CurrentAccountBalance&docFormKey=88888888&hideReturnLink=true",
+        "linkType": "kfs",
+        "permission": {
+          "templateNamespace": "KR-NS",
+          "templateName": "Use Screen",
```

Comment:
I think I was trying to be consistent with the other balance inquiries, but they are not straight lookups like this, so you're probably right that a "Look Up Records" permission would probably be better. (FWIW, this customization will be probably be removed soon as long as we can change the label since this lookup is in the base code now.)